Add description metadata (#1659)
ezsystems/developer-documentation: docs/guide/search/search.md

---

> description: Ibexa DXP search functionalities allows working with three search engines and using search API to run complex and precise question of content and products.

Author: ```suggestion
description: Ibexa DXP search functionalities allow working with three search engines and using search API to run complex and precise queries about content and products.
```
should have been 'queries' from the start, not sure how 'questions' got there
